loader/github.go

```diff
@@ -2,9 +2,14 @@ package loader
 
 import "github.com/sirupsen/logrus"
 
-func github(_ logrus.FieldLogger, _ string, parts []string) (string, error) {
+func github(logger logrus.FieldLogger, specifier string, parts []string) (string, error) {
 	username := parts[0]
 	repo := parts[1]
 	filepath := parts[2]
-	return "https://raw.githubusercontent.com/" + username + "/" + repo + "/master/" + filepath, nil
+	realURL := "https://raw.githubusercontent.com/" + username + "/" + repo + "/master/" + filepath
+	logger.Warnf(magicURLsDeprecationWarning, specifier, "github", realURL)
+	return realURL, nil
 }
+
+const magicURLsDeprecationWarning = "Specifier %q resolved to use a non-conventional %s loader. " +
+	"That loader is deprecated and will be removed in v0.53.0. Please use the real URL %q instead."
```

Comment:
```suggestion
const magicURLsDeprecationWarning = "Specifier %q was resolved by a non-conventional %s loader. " +
	"The used %q loader is deprecated and will be removed in k6 v0.53.0. Please, use the real URL %q instead."
```
In any case, my feeling is that `non-conventional` isn't clear enough information. Is there a de-facto list of _conventional loaders_ for JavaScript that I'm not aware of?
I don't think we define somewhere the k6 _conventional_ (supported?) loaders. We should explicit them here or in the documentation.

I don't know if this is

Reply:
There isn't - and there is no documentation about cdnjs and github loaders for years at this point - we did drop it from every documentation like 3+ years ago.

I do agree - but IMO us making this message be more specific likely will have no benefit. I have not seen anyone use the github ones *ever* and the cdnjs ones seems to be used *very* rarely. 

So my expectation is most people will never see this message. 

If you have a concrete simple fix - I am all for it, but my only other proposal was "magic" (with quotes) and I think that is even less obvious. 

I would also argue that most people will agree that pure paths and URLs are the "conventional" specifiers even if the specification has left this completely unspecified :wink: :wink: